helpers/make_template.py

```diff
@@ -50,15 +53,16 @@ def main (options_file, template_file):
                     entered_restart_section=True
 
                 key = (l.split("=")[0]).strip().lower()
-                if key == "restart_file":
-                    if verbose: print("Writing restart_file line")
-                    tmpl.write('    restart_file="__RESTART_FILE__"\n')
-                elif key == "restart_date":
+                ## as long as the default (i.e. restart_file="restart/icar_rst_" ) is used, no need to change.
+                # if key == "restart_file":  
+                #     if verbose: print("Writing restart_file line")
+                #     tmpl.write('    restart_file="__RESTART_FILE__"\n')
+                if key == "restart_date":
                     if verbose: print("Writing restart_date line")
                     tmpl.write('    restart_date= __RESTART_DATE__,\n')
-                elif key == "restart_step":
-                    if verbose: print("Removing restart_step line")
-                    pass
+                # elif key == "restart_step":
```

Comment:
I think restart_step is no longer used and should just be removed entirely (rather than commented out)